tests/template_tests/test_origin.py

```diff
@@ -0,0 +1,24 @@
+from unittest import TestCase
+
+from django.template import Engine
+
+from .utils import TEMPLATE_DIR
+
+
+class OriginTestCase(TestCase):
```

Comment:
Oops sorry, I missed the import somehow and thought you were using `django.test.TestCase`